docs/sphinx/source/whatsnew/v0.11.0.rst

```diff
@@ -21,6 +21,8 @@ Enhancements
   shade perpendicular to ``axis_azimuth``. The function is applicable to both
   fixed-tilt and one-axis tracking systems.
   (:issue:`1689`, :pull:`1725`, :pull:`1962`)
+* Updated `~pvlib.iotools.MIDC_VARIABLE_MAP`~ to reflect
+  changes in instrumentation. (:pull:`2006`)
```

Comment:
I would call this a Breaking Change, as user code that works with pvlib 0.10.5 won't work with 0.11.0